content/en/synthetics/guide/api_test_timing_variations.md

```diff
@@ -0,0 +1,119 @@
+
+These metrics are only collected if the request goes over HTTPS, and not HTTP. 
+
+In case of any redirection, the SSL handshake timing only corresponds to  the last request.
+
+
+### Time to First Byte
+
+The `synthetics.http.firstbyte.time` metric measures the time between the moment the connection was established and the moment Datadog client received the first byte of the response. Any time spent sending data in the request is included in this timing. 
+
+In case of any redirection, the time to first byte only corresponds to the last request.
+
+### Download time
+
+The `synthetics.http.download.time` metric measures the time between the moment Datadog client receives the first byte of the response and the moment it finishes downloading the whole response. Generally, the bigger the response body, the higher this timing will be. 
+
+In the case the response does not have a body, this timing is null.
+
+In case of any redirections, the download time only corresponds to the last request.
+
+### Total response time
+
+The `*.response.time`  measures the total time between the moment Synthetics starts the request and the moment it finishes it. The response time is the sum of all network timings. For instance for an HTTP test with no redirections on an HTTPS endpoint, `synthetics.http.response.time = synthetics.http.dns.time + synthetics.http.connect.time+synthetics.http.ssl.time + synthetics.http.download.time`.
+
+## Timing variations
+ 
+Variations in API tests network timing metrics can occur when there is a bottleneck or delay during any of the several stages of the request, from the redirection to the download of the response body. Usually, it is helpful to identify the following : 
+
+- If the variation is observed as a general trend, or as a sudden spike
+- If the variation only occurs at specific stage of the request, for instance only for the DNS timings
+- If the impacted Synthetics test is running from multiple locations, whether the variation is localised to a single location, or if it is widespread
+- If the variation only occurs for a single URL, domain, subdomain, or if it is impacting all tests.
+
+
+
+For each timing metric we measure, variations can usually be explained by several factors:
+
+### Redirection time
+Given this timing is the sum of all redirects in a request, variations at any stage of the HTTP request, from the DNS resolution to the download can noticeably increase the redirection timing. For example any delay in DNS resolution would impact the redirection timing as redirections require API tests to resolve multiple domains.
+
+### DNS resolution time
+An increased DNS resolution time could be due to additional latency from authoritative servers.
+
+### TCP connection time
+Variations of the TCP handshake can be due to the load of the network and server, the size of the request and response messages, and the distance between the Synthetics managed/[private location][6] and the server.
```

Comment:
```suggestion
Variations of the TCP handshake can occur because of the network and server load, the size of the request and response messages, and the distance between the Synthetics managed and [private location][6] and the server.
```